src/mumble/SettingsKeys.h

```diff
@@ -169,7 +169,8 @@ const SettingsKey CHANNEL_EXPANSION_MODE_KEY           = { "channel_expansion_mo
 const SettingsKey CHANNEL_DRAG_MODE_KEY                = { "channel_drag_mode" };
 const SettingsKey USER_DRAG_MODE_KEY                   = { "user_drag_mode" };
 const SettingsKey ALWAYS_ON_TOP_KEY                    = { "always_on_top" };
-const SettingsKey ASK_ON_QUIT_KEY                      = { "ask_on_quit" };
+const SettingsKey ASK_ON_QUIT_KEY					   = { "ask_on_quit" };
```

Comment:
Actually, I thought about this one again. We don't need to keep this old entry after all, as the code in `Settings::legacyLoad` has to hard-code the settings-key anyway (or else the code would get more complicated than necessary). Thus, this old key can be deleted after all